File: VB/S92027/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraBars
Imports DevExpress.XtraEditors

Namespace S92027
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub OnShowMessaageClick(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			XtraMessageBox.Show("Test", "DXSample")
		End Sub

		Private Sub OnCloseClick(ByVal sender As Object, ByVal e As EventArgs) Handles mySimpleButton2.Click
			Close()
		End Sub
	End Class
End Namespace

File: CS/S92027/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraBars;
using DevExpress.XtraEditors;

namespace S92027 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void OnShowMessaageClick(object sender, EventArgs e) {
            XtraMessageBox.Show("Test", "DXSample");
        }

        private void OnCloseClick(object sender, EventArgs e) {
            Close();
        }
    }
}